package.json

```diff
@@ -3,7 +3,8 @@
   "name": "uppy-build",
   "description": "Extensible JavaScript file upload widget with support for drag&drop, resumable uploads, previews, restrictions, file processing/encoding, remote providers like Instagram, Dropbox, Google Drive, S3 and more :dog:",
   "lint-staged": {
-    "*.js": "eslint"
```

Comment:
Not really related, but may I ask you to add [bin config](https://docs.npmjs.com/files/package.json#bin) like so?

```json
...
"bin": {
  "companion" : "./lib/standalone/start-server.js"
},
...
```

or similar?

That way a `companion` file would be dropped in `node_modules/.bin` and folks & machines could just run `companion` (if they have it in their PATH, or start it from npm scripts), vs having to do some research to get at `./lib/standalone/start-server.js`

Reply:
@kvz we have that [here](https://github.com/transloadit/uppy/blob/1a2e00d766478dee773439ce683377ae13d21dbe/packages/%40uppy/companion/package.json#L28-L30) already.

Reply:
Ohhhh oops, sorry for overlooking!